scrapers/covid-19/historical.js

```diff
@@ -90,6 +113,13 @@ const historicalV2 = async (keys, redis) => {
  */
 const getHistoricalDataV2 = (data, lastdays = 30) => {
 	lastdays = stringUtils.getLastDays(lastdays);
+	// Removing data inserted for edge case of Canada
+	for (let i = 0; i < data.length; i++) {
+		if (data[i].country.toLowerCase() === 'canada' && data[i].province.toLowerCase() === 'recovered-aggregate') {
+			data.splice(i, 1);
+			break;
+		}
+	}
```

Comment:
Why are we removing it?

Reply:
If I understood correctly, ` getHistoricalDataV2` is invoked after hitting the endpoints `/v3/covid-19/historical` and `/v3/covid-19/historical/all` which returns all data for the last 30 days. Since data for countries with `Provinces/Regions` are disaggregated by `Provinces/Regions` then the fake province inserted primarily for holding cumulative aggregate recovered data for Canada will also be included in the returned data if it is not removed.  I hope  `data` is a deep copy of what has been scraped.